base/migrations/0002_alter_image_image.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.3 on 2023-07-07 12:49
+
+from django.db import migrations, models
```

Comment:
alter_image_image?

Reply:
why 0001_initial? why 0002?

Reply:
> alter_image_image?

@sagynbek001 It was created and named automatically when I have changed database.

Reply:
> why 0001_initial? why 0002?

I did not named them. When I wrote "python3 manage.py makemigrations" in ubuntu to update database, they were created by program

Reply:
OKay, but they sound weird. Maybe there is a flag when creating to put a chosen name. Take look next time.